vars/cloudFoundryDeploy.groovy

```diff
@@ -117,14 +118,16 @@ def deployCfNative (config) {
         passwordVariable: 'password',
         usernameVariable: 'username'
     )]) {
-        def deployCommand = 'push'
+        def deployCommand = getCfNativeDeployCommand(config)
```

Comment:
The name `getCfNativeDeployCommand` does not really reveal what this method does. How about `selectCfDeployCommandForDeployType` or similar?

Reply:
done